bobtemplates/plone_addon/src/+package.namespace+/+package.name+/configure.zcml.bob

```diff
@@ -17,10 +17,29 @@
   <plone:static
       directory="theme"
       type="theme"
-      name="{{{ package.dottedname }}}"
+      name="{{{ package.name }}}"
+      />
+
+  <genericsetup:registerProfile
+      name="default"
+      title="Plone Theme: {{{ package.name }}}"
```

Comment:
Using just the `name` instead of `dottedname` here doesn't work so well if you have a different naming convention. I generally name themes `product.theme`, and in this case, the entry in the Add-ons panel just looked like "Plone Theme: theme".  I would recommend using `dottedname` here, like on the uninstall profile.